src/renderer/init.js

```diff
@@ -103,6 +103,11 @@ async function init() {
   if (isMainWindow) {
     webFrame.setVisualZoomLevelLimits(1, 1);
 
+    const matcher = window.matchMedia("(prefers-color-scheme: dark)");
+    const updateOSTheme = () => store.dispatch(setOSDarkMode(matcher.matches));
+    matcher.addListener(updateOSTheme);
+    updateOSTheme();
```

Comment:
here typically, i think we need to prevent a dispatch() but just make the "initial state" having the current OS value.

Reply:
I don't get it

Reply:
got it